lib/pure/nativesockets.nim

```diff
@@ -231,17 +233,17 @@ proc ntohs*(x: int16): int16 =
   when cpuEndian == bigEndian: result = x
   else: result = (x shr 8'i16) or (x shl 8'i16)
 
-proc htonl*(x: int32): int32 =
+template htonl*(x: int32): expr =
```

Comment:
Maybe, but I remember speaking with @Araq and I think using templates is encouraged for doing simple inlining in Nim, and `proc {.inline.}` is for some more complicated cases.